src/js/components/calendar.jsx

```diff
@@ -19,13 +19,20 @@ export default class Calendar extends React.Component {
       type: 'hidden',
     };
 
+    const timeslotProps = {
+      format: 'h',
+      showFormat: 'h:mm A',
+    };
+
     super(props);
 
     this.inputProps = {
       startDate: Object.assign({}, startDateInputProps, this.props.startDateInputProps),
       endDate: Object.assign({}, endDateInputProps, this.props.endDateInputProps),
     };
 
+    this.timeslotProps = Object.assign({}, timeslotProps, this.props.timeslotProps);
```

Comment:
Whenever you do something like this, make sure to handle the same thing into `componentDidUpdate` or `componentWillReceiveProps`, as you don't know if users might change the `timelotProps` prop.

For reference, check `MonthComponent`, in which if the startDate is update, the `currentWeek` state value is updated.